src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -167,6 +173,9 @@
     /** A string value of 1. */
     private static final String ONE_STRING_VALUE = "1";
 
+    /** Default distance between tab stops. */
+    private static final String DEFAULT_TAB_WIDTH = "8";
```

Comment:
is there any our previous discussion on why 8  is default?
in our code we do not use tabs, but tabs usually mean indent step.
I did not found statistics in web by word more and more going to indent size of 2 or 4.
Lets make it 4.

Reply:
I took as example `TreeWalker.DEFAULT_TAB_WIDTH` which is 8 by default

Reply:
ok, make sense.